### ISSUE TITLE
Is there anyone train your data successfully？

### ISSUE THREAD

**Issue by wanghuite** [balancap/SSD-Tensorflow#206]
I change a lit the code for my two-type detection tasks,but my loss is around 10 for a long time. Is there any one can reduce the loss much lower?

**Comment by phinoo** [comment]
hello  do you have a wechat or QQ,we can talk abaout this project

**Comment by edisonchensy** [comment]
i have some questions on training my data, can i add your qq to ask?

**Comment by yangtx2009** [comment]
can I add your wechat for asking some questions about the training?

**Comment by edisonchensy** [comment]
sure，it’s edison6900




| |
EDC
邮箱：shandianball@126.com
|

签名由 网易邮箱大师 定制

On 05/31/2018 17:12, yangtx2009 wrote:

can I add your wechat for asking some questions about the training?
